File: c++/HouseSection.cpp
// Author : Mustafa Mulla
// Roll No : 27
// Title : Goa legistalive assembly
// Start date : 15/07/2024
// Modified date :  22/07/2024
// Description : Goa Legislative Assembly's various sections
#include "HouseSection.h"

HouseSection::HouseSection() : Goa_Legislative("House"){
    subsections.push_back(Subsection("Session Information"));
    subsections.push_back(Subsection("Question List"));
    subsections.push_back(Subsection("Assembly Bills"));
    subsections.push_back(Subsection("Papers laid in the house"));
}

std::vector<Subsection> HouseSection::getSubsections(){
    return subsections;
}

File: python/house_section.py
# Author : Mustafa Mulla
# Roll No : 27
from typing import List
from goa_legislative import GoaLegislative
from subsection import Subsection

class HouseSection(GoaLegislative):
    def __init__(self):
        super().__init__("House")
        self._subsections = [
            Subsection("Session Information"),
            Subsection("Question List"),
            Subsection("Assembly Bills"),
            Subsection("Papers laid in the house")
        ]

    def get_subsections(self) -> List[Subsection]:
        return self._subsections
